_posts/2019-01-25-building-apps-with-plug-router.md

```diff
@@ -0,0 +1,345 @@
+---
+author: Sean Callan
+categories: general
+date: 2019-01-25
+layout: post
+title: Building web apps with Plug.Router
+---
+
+When it comes to building a web application with Elixir most people will immediately reach for Phoenix, but did you know Plug.Router is just as vaiable an option and sometimes even faster?
```

Comment:
run-ons and a few typos, would alter to:
```
When it comes to building a web application with Elixir many people will immediately reach for Phoenix.
However, did you know `Plug.Router` is just as viable an option?
Sometimes, it can be even faster.
```